File: C_Programming/Study/Q3_Assignment9.c
#include <stdio.h>

///////////////////////////////////////////////////////////////////////////////////////////////////////////
//
// Function Name :  CountRange
// Output:          
// Description:     Assignment_9: Q.3: Write a program which accept Number & return the count of Digits in between 3 & 7. 
//                                     
// Date:            30/05/2026
// Author:          Shital Ajit Nikam   
//
///////////////////////////////////////////////////////////////////////////////////////////////////////////


int CountRange(int iNo)
{
    int iDigit = 0;
    int iCnt = 0;

   
    while(iNo != 0)
    {
        iDigit = iNo % 10;
        if(iDigit > 3 && iDigit < 7)
        {
            iCnt++;
        }

        iNo = iNo /10;
    }
    
    return  iCnt;
}

int main()
{
    int iValue = 0;
    int iRet = 0;
        
    printf("Enter Number: \n");
    scanf("%d", &iValue);

    iRet = CountRange(iValue);

    printf("Count of Digits in between 3 & 7 is : %d",iRet);

    return 0;
}

/////////////////////////////////////////////////////////////////////////////////////
//Time Complexity : O(N)  where N = length of number
/////////////////////////////////////////////////////////////////////////////////////
// Input : 2395
// Output : 1  
// Input : 1018
// Output : 0
// 
/////////////////////////////////////////////////////////////////////////////////////

File: Python_Programming/Competative/Q3_Assignment9.py
#//////////////////////////////////////////////////////////////////////////////////////////////////////////
# Function Name :  Square
#        
# Description:     Assignment_9: Q.3: Write a program which accept one number & prints square of that number.
# Input  :         5
# Output :         25
#                                     
# Date:            30/06/2026
# Author:          Shital Ajit Nikam   
#
#//////////////////////////////////////////////////////////////////////////////////////////////////////////

def Square(No1):
    Square = 0
    Square = No1 ** 2               #Square of number (No1 ^ 2)
    return Square
    
def main():
    value1 = int(input("Enter the number: "))
        
    Ret = Square(value1)   
    
    print(f"Square of {value1} is {Ret}")   
     
    
if __name__ == "__main__":
    main()